cyhy_commander/commander.py

```diff
@@ -24,10 +24,12 @@
 from collections import defaultdict
 import logging
 import os
+import Queue
```

Comment:
Should this be 
```suggestion
from queue import Queue
```

See the [Python documentation](https://docs.python.org/3/library/queue.html).

Reply:
We are running under Python 2 and per the [library documentation](https://docs.python.org/2.7/library/queue.html):

> Note: The [`Queue`](https://docs.python.org/2.7/library/queue.html#module-Queue) module has been renamed to `queue` in Python 3. The [2to3](https://docs.python.org/2.7/glossary.html#term-2to3) tool will automatically adapt imports when converting your sources to Python 3.

I am intentionally importing the whole library because I need `Queue.Queue()` and the `Queue.Empty` exception.

Reply:
Ah, right.  I forgot that this was Python 2.